pkg/sfu/downtrack.go

```diff
@@ -298,9 +298,6 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 	d.bound.Store(true)
 	d.bindLock.Unlock()
 
-	if d.onMaxLayerChanged != nil {
-		d.onMaxLayerChanged(d, d.MaxLayers().Spatial)
-	}
```

Comment:
This is what i believe was causing the race. Explained in PR commit notes above as to why this is not needed.